src/videojs-contrib-hls.js

```diff
@@ -134,6 +133,46 @@ export default class HlsHandler extends Component {
       externHls: Hls
     });
 
+    this.masterPlaylistController_.on('loadedmetadata', () => {
+      let audioTracks = this.tech_.audioTracks();
```

Comment:
Not to picky at all, they will actually have the same value here, we can just re-use that old variable, as they will both return the same video.js tracklist (it never gets deleted or re-created just cleared out)